modules/hcp-get-ocp-channel.adoc

```diff
@@ -2,134 +2,82 @@
-    url: https://access.redhat.com/errata/RHSA-2024:5422
-    version: 4.16.8
-  - channels:
-    - candidate-4.16
-    - candidate-4.17
-    - eus-4.16
-    - fast-4.16
-    - stable-4.16
-    image: quay.io/openshift-release-dev/ocp-release@sha256:e4102eb226130117a0775a83769fe8edb029f0a17b6cbca98a682e3f1225d6b7
-    url: https://access.redhat.com/errata/RHSA-2024:4965
-    version: 4.16.6
-  - channels:
-    - candidate-4.16
-    - candidate-4.17
-    - eus-4.16
-    - fast-4.16
-    - stable-4.16
-    image: quay.io/openshift-release-dev/ocp-release@sha256:f828eda3eaac179e9463ec7b1ed6baeba2cd5bd3f1dd56655796c86260db819b
-    url: https://access.redhat.com/errata/RHBA-2024:4855
-    version: 4.16.5
-  conditionalUpdates:
-  - conditions:
-    - lastTransitionTime: "2024-09-23T22:33:38Z"
-      message: |-
-        Could not evaluate exposure to update risk SRIOVFailedToConfigureVF (creating PromQL round-tripper: unable to load specified CA cert /etc/tls/service-ca/service-ca.crt: open /etc/tls/service-ca/service-ca.crt: no such file or directory)
-          SRIOVFailedToConfigureVF description: OCP Versions 4.14.34, 4.15.25, 4.16.7 and ALL subsequent versions include kernel datastructure changes which are not compatible with older versions of the SR-IOV operator. Please update SR-IOV operator to versions dated 20240826 or newer before updating OCP.
-          SRIOVFailedToConfigureVF URL: https://issues.redhat.com/browse/NHE-1171
-      reason: EvaluationFailed
-      status: Unknown
-      type: Recommended
-    release:
-      channels:
-      - candidate-4.16
-      - candidate-4.17
-      - eus-4.16
-      - fast-4.16
-      - stable-4.16
-      image: quay.io/openshift-release-dev/ocp-release@sha256:fb321a3f50596b43704dbbed2e51fdefd7a7fd488ee99655d03784d0cd02283f
-      url: https://access.redhat.com/errata/RHSA-2024:5107
-      version: 4.16.7
-    risks:
-    - matchingRules:
-      - promql:
-          promql: |
-            group(csv_succeeded{_id="d6d42268-7dff-4d37-92cf-691bd2d42f41", name=~"sriov-network-operator[.].*"})
-            or
-            0 * group(csv_count{_id="d6d42268-7dff-4d37-92cf-691bd2d42f41"})
-        type: PromQL
-      message: OCP Versions 4.14.34, 4.15.25, 4.16.7 and ALL subsequent versions
-        include kernel datastructure changes which are not compatible with older
-        versions of the SR-IOV operator. Please update SR-IOV operator to versions
-        dated 20240826 or newer before updating OCP.
-      name: SRIOVFailedToConfigureVF
-      url: https://issues.redhat.com/browse/NHE-1171
+.. Optional: To retrieve more detailed information about the status of your update, run the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade status
+----
++
+For more information about how to use the `oc adm upgrade status` command, see "Gathering cluster update status using oc adm upgrade status (Technology Preview)".
```

Comment:
while it's tech-preview, you'll also need to set environment variables to get `status` to work.  If you want to inline the `oc adm upgrade status`, you may also want to inline that environment variable advice.  Alternatively, you can probably drop the inline `oc adm upgrade status` and just link over to that section of the standalone update docs, and folks can find both the environment variable information and the subsequent `oc adm upgrade status` invocation over there.